sdk/python/feast/data_source.py

```diff
@@ -365,6 +367,100 @@ def to_proto(self) -> DataSourceProto.KinesisOptions:
         return kinesis_options_proto
 
 
+class SqlServerOptions:
```

Comment:
Data sources are now here https://github.com/feast-dev/feast/tree/master/sdk/python/feast/infra/offline_stores

Reply:
Our preference would be for you to try and implement the offline store and data source externally (separate Python package) and use custom data sources for configuration (as opposed to making changes to the core protos). Would that work? https://github.com/feast-dev/feast/pull/1713